### ISSUE TITLE
amavisd-new is restarting all the time

### ISSUE THREAD

**Issue by lejmr** [lejmr/iredmail-docker#53]
I can see this in messages log. 

```May 13 07:34:11 mail root: DEBUG Starting amavisd-new
May 13 07:34:12 mail root: DEBUG Starting amavisd-new
May 13 07:34:14 mail root: DEBUG Starting amavisd-new
May 13 07:34:16 mail root: DEBUG Starting amavisd-new
May 13 07:34:18 mail root: DEBUG Starting amavisd-new
May 13 07:34:20 mail root: DEBUG Starting amavisd-new```

It seems amavisd is not actually restarting because exec command stops as it finds other pid.

**Comment by nizovtsevnv** [comment]
The same thing practically:
```
Dec 13 18:32:37 mail root: DEBUG Starting amavisd-new
The amavisd daemon is already running, PID: [25782]
*** Starting amavis..
Dec 13 18:32:39 mail root: DEBUG Starting amavisd-new
The amavisd daemon is already running, PID: [25782]
*** Starting amavis..
Dec 13 18:32:40 mail root: DEBUG Starting amavisd-new
The amavisd daemon is already running, PID: [25782]
*** Starting amavis..
Dec 13 18:32:42 mail root: DEBUG Starting amavisd-new
The amavisd daemon is already running, PID: [25782]
```

This issue eats much CPU...

**Comment by nizovtsevnv** [comment]
For resolving you can just return zero code in /etc/service/amavis/run or stop&start, or restart amavis service.

**Comment by lejmr** [comment]
Should be resolved with version 1.0
